http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -856,6 +856,13 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
 }
 
 object RuntimeServiceInterp {
+
+  private def getToolFromImages(clusterImages: Set[RuntimeImage]): Option[Tool] =
```

Comment:
can u add a few unit tests for this function...u can change `private` to `private[service]` so that it can be called in unit tests